rotkehlchen/tests/unit/decoders/test_zksync.py

```diff
@@ -104,3 +104,88 @@ def test_zksync_lite_deposit(database, ethereum_inquirer, ethereum_accounts):
         ),
     ]
     assert expected_events == events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x9531C059098e3d194fF87FebB587aB07B30B1306']])
+def test_zksync_lite_withdrawal(database, ethereum_inquirer, ethereum_accounts):
+    """Test a transaction with the Withdrawal event"""
+    tx_hex = deserialize_evm_tx_hash('0x234407968b9a688be3fb37cf7ff8ef3b4168d6cd85ec45b8344bb2a88832f982')  # noqa: E501
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    user_address = ethereum_accounts[0]
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hex,
+    )
+    timestamp, eth_str, dai_str, pan_str, usdc_str, usdt_str = TimestampMS(1604326368000), '1.4437093', '1691.92749999', '1586.6', '57.25', '15.2'  # noqa: E501
+    a_pan = Asset('eip155:1/erc20:0xD56daC73A4d6766464b38ec6D91eB45Ce7457c44')
```

Comment:
I just noticed that this variable is used in only 1 place so instead of crating a variable just replace its usage by `Asset('eip155:1/erc20:0xD56daC73A4d6766464b38ec6D91eB45Ce7457c44')`